static/js/tools/timeline_util.ts

```diff
@@ -170,6 +176,28 @@ function getStatsVarInfo(dcids) {
   });
 }
 
+function getStatsVar(dcids: string[]) {
+  if (dcids.length === 0) {
+    return Promise.resolve([]);
+  }
+  const promises = [];
+  for (const dcid of dcids) {
+    promises.push(
+      axios.get("/api/place/statsvars/" + dcid).then((resp) => {
+        return resp.data;
+      })
+    );
+  }
```

Comment:
can you add a todo here to read the set from server side

Reply:
done.